coralproject/cay
# remove react-redbox transformation

## What does this PR do?

`react-redbox` transformer would swallow errors and make console output useless. This was fine most of the time, unless the error only flashed for a second.
## How do I test this PR?
- make an error in a component's `render()` method, like an undefined variable or something
- see that there's a line number now that you can click through to the Source tab in devtools

@coralproject/frontend
